### PR TITLE
Issue6: Confirmation pages

### LINKED ISSUE #6: [FE] Confirmation Pages
Account Created Confirmation
- Create Page
- "Return to login" button navigates to log in page
Design: https://www.figma.com/file/cN9xGpLOjwApmLoQXTgkqB/H4I%3A-Stand-Strong-%2B-RISE-Crisis-Line-Volunteer-Management-App?node-id=344%3A5514

Password Changed Confirmation
- Create page
- "Return to login" button navigates to log in page
Design: https://www.figma.com/file/cN9xGpLOjwApmLoQXTgkqB/H4I%3A-Stand-Strong-%2B-RISE-Crisis-Line-Volunteer-Management-App?node-id=437%3A6182

### PR DESCRIPTION
**Pull request by thebarrettlo**
Closes #6. Separated password change confirmation and new account creation confirmation pages into separate pages. Passing form data from the new account creation to the confirmation page took lots of time, but it now works using a react-router `<Navigate state={} />` and react-router `const location = useLocation();` on the receiving end.

### CODE REVIEW

File: frontend/src/components/ChangePasswordPage/ForgotPassword_Email.js [hunk 1]
```diff
@@ -45,41 +45,44 @@ const ForgotPassword_Email = () => {
     }
 
     return(
-        <CenterBox textAlign="center">
-            <LogoWithBack back="/"/>
+        <CenterBox>
+           <Center>
+               <Box textAlign="center">
```
Comment (VincentPheng) [inline frontend/src/components/ChangePasswordPage/ForgotPassword_Email.js:50]: Remove box and center component

File: frontend/src/components/ChangePasswordPage/ForgotPassword_Email.js [hunk 2]
```diff
@@ -45,41 +45,44 @@ const ForgotPassword_Email = () => {
     }
 
     return(
-        <CenterBox textAlign="center">
-            <LogoWithBack back="/"/>
+        <CenterBox>
```
Comment (VincentPheng) [inline frontend/src/components/ChangePasswordPage/ForgotPassword_Email.js:48]: include textAlign="center" on centerbox again

**Review by VincentPheng — CHANGES_REQUESTED** [review]

**Comment by VincentPheng** [comment]
Fix conflicts

**Comment by thebarrettlo** [comment]
Bad commit history. See #38 for a working PR of issue 6.

**Pull request closed without merging**